control-plane-operator/controllers/hostedcontrolplane/v2/dnsoperator/component.go

```diff
@@ -0,0 +1,49 @@
+package dnsoperator
+
+import (
+	oapiv2 "github.com/openshift/hypershift/control-plane-operator/controllers/hostedcontrolplane/v2/oapi"
+	component "github.com/openshift/hypershift/support/controlplane-component"
+	"github.com/openshift/hypershift/support/util"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	ComponentName = "dns-operator"
+)
+
+var _ component.ComponentOptions = &dnsOperator{}
+
+type dnsOperator struct {
+}
+
+// IsRequestServing implements controlplanecomponent.ComponentOptions.
+func (d *dnsOperator) IsRequestServing() bool {
+	return false
+}
+
+// MultiZoneSpread implements controlplanecomponent.ComponentOptions.
+func (d *dnsOperator) MultiZoneSpread() bool {
+	return true
```

Comment:
I think this should be false according to the v1 code here 
https://github.com/openshift/hypershift/blob/2753dcc8cfb7591a61d64d26766526f844cebffa/control-plane-operator/controllers/hostedcontrolplane/dnsoperator/dnsoperator.go#L75

Reply:
I don't think it makes any difference. this logic is from the deploymentConfig. 

> maybe we should tied both semantics together

makes sense